doc/nrf/gs_updating.rst

```diff
@@ -14,6 +14,20 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _gs_updating_in_vsc:
+
+Updating |VSC|
+*****************
+
+Visual Studio Code checks for extension updates and automatically installs them when they are available.
+After an extension is updated, VS Code prompts you to reload the application.
+If you disabled automatic updates, open the :guilabel:`Extensions` tab and locate the |VSC| extension.
```

Comment:
What do you think, does this now need a paragraph break?
Not because of length, but as a sort of thought reset.

Reply:
Yes, I agree with @wiba-nordic .

Reply:
Updated with a paragraph break.